cellpack/autopack/Environment.py

```diff
@@ -999,8 +1100,9 @@ def restoreGridFromFile(self, gridFileName):
         aSurfaceGrids = []
         f = open(gridFileName, "rb")
         self.readArraysFromFile(f)
-        for compartment in self.compartments:
-            compartment.readGridFromFile(f)
+        for ct, compartment in enumerate(self.compartments):
+            # compartment.readGridFromFile(f)
```

Comment:
delete?

Reply:
Verified the saving and loading, the updated functions should work now